src/__tests__/__snapshots__/profilingCharts-test.js.snap

```diff
@@ -83,8 +83,8 @@ Object {
         "actualDuration": 0,
         "didRender": true,
         "id": 5,
-        "label": "Memo(Child) key=\\"third\\" (<0.1ms of <0.1ms)",
-        "name": "Memo(Child)",
+        "label": "Child key=\\"third\\" (<0.1ms of <0.1ms)",
```

Comment:
This raises an interesting question. Should we show the badge in the profiling chart views too? e.g. "Child [memo]"?

Reply:
Yes definitely we should do that. 
Should the badge “Memo” or “ForwardRef” be in text format as you mentioned here? ( Child [memo] )
Or in the firefox like badge format?